src/doc/unstable-book/src/compiler-flags/regparm.md

```diff
@@ -0,0 +1,20 @@
+# `regparm`
+
+The tracking issue for this feature is: https://github.com/rust-lang/rust/issues/131749.
+
+------------------------
+
+Option -Zregparm=N causes the compiler to pass N arguments
+in registers EAX, EDX, and ECX instead of on the stack.
```

Comment:
With the `extern "Rust"` support removed, we should be explicit this affects a limited number of ABIs. I sincerely doubt it should affect `extern "x86-interrupt"`, for instance!

```suggestion
in registers EAX, EDX, and ECX instead of on the stack for "C", "cdecl", and "stdcall" fn.
```

If we want, we can be more vague:
```suggestion
in registers EAX, EDX, and ECX instead of on the stack for some non-Rust ABIs.
```